Examples/ServiceExamples/Scripts/ExampleConversation.cs

```diff
@@ -57,6 +57,40 @@ private void OnMessageWithOnlyInput(MessageResponse resp, string customData)
         foreach (string txt in resp.output.text)
           Debug.Log("Message Only output: " + txt);
 
+      if (resp.context != null)
```

Comment:
I've done this for `Conversation`. I need some time to implement this in all other services. This was not built into the SDK but it's not that difficult to implement. I created an issue in dev-experience (#2131)